packages/amplication-data-service-generator/tsconfig.json

```diff
@@ -6,7 +6,7 @@
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "target": "es2017",
-    "lib": ["es2020"],
+    "lib": ["es2020", "dom"],
```

Comment:
@yuval-hazaz this is not working without the "dom".
@mshidlov added it. I am not sure why it worked before without it.